regulations/views/comment.py

```diff
@@ -43,9 +43,34 @@ def upload_proxy(request):
     })
 
 
+def download_proxy(request):
+    """Generate a temporary URL to GET an attachment."""
+    key = request.GET.get('key')
+    name = request.GET.get('name')
+    if key is None or name is None:
+        return JsonResponse(
+            {'message': 'Must provide key and name'},
+            status=400,
+        )
+    s3 = tasks.make_s3_client()
+    disposition = 'attachment; filename="{}"'.format(name)
+    url = s3.generate_presigned_url(
+        ClientMethod='get_object',
+        Params={
+            'ResponseContentDisposition': disposition,
+            'Bucket': settings.ATTACHMENT_BUCKET,
+            'Key': key,
+        },
+    )
+    return JsonResponse({'url': url})
```

Comment:
This is a tad concerning as we're no longer tying the ability to preview a document with the "session" that created it. I'm pretty sure it's not actually an issue as, our "key"s are ~300bits of randomness, but I wonder if there's not a simple way to tie this a bit tighter. For example, would it make sense to issue both a preview _and_ upload URL when the user first requests to upload the file? Presumably the URLs have a timeout, which would make that less useful.